python/paddle/v2/parameters.py

```diff
@@ -42,9 +43,26 @@ def create(layers):
 
 class Parameters(object):
     """
-    Parameters is a dictionary contains Paddle's parameter. The key of
-    Parameters is the name of parameter. The value of Parameters is a plain
-    :code:`numpy.ndarry` .
+    `Parameters` manages all the learnable parameters in a neural network.
+    It stores parameters' information in an OrderedDict, key of which is
+    the name of a parameter, and value related to a key is a parameter's
+    configuration, such as initialization mean and std, its size, whether it is
+    a static parameter, and so on.
+
+    :param __param_conf__: this member stores the configurations of learnable
+        parameters in a network in an OrderedDict. The parameters are added by
+        following their creation order in the neural network one by one:
+        parameters of the previous layers in a network are careted first.
+        When a user iterates over this dict, he can visit parameters in the
```

Comment:
You can visit the parameters from bottom to top by iterating over this dict.

Reply:
done.